pkg/healthcheck/log.go

```diff
@@ -0,0 +1,261 @@
+/*
```

Comment:
Nit: calling it log.go is not very accurate. May be we can split the log specific methods from the containerd metadata update methods into their own files.